gnoboard/src/native_modules/types.ts

```diff
@@ -10,4 +10,10 @@ export type GnoConfig = {
   Mnemonic: string;
 };
 
+export type NetworkMetainfo = {
+  chainId: string;
+  chainName: string;
+  gnoUrl: string;
```

Comment:
For me, "gnoUrl" is a confusing name. A URL has something like "http://", but the Gno remote doesn't have this. Maybe "gnoAddress"?